### PR TITLE
Update manage-policies.html.md

### PR DESCRIPTION
**Pull request by susiesolis**
## Labels

- [ ] inaccuracy
- [ ] clarification
- [ ] new docs
- [ ] cosmetic bug - fixing broken text or markup
- [ ] enhancement - changing the website's behavior/layout

### CODE REVIEW

**Comment by vancluever** [comment]
Hey @susiesolis, I think this is only really necessary when using GitHub. Maybe add a note mentioning to do this when using GitHub?

**Comment by vancluever** [comment]
@susiesolis can you also add how someone views a raw file within a particular page in GH too?

**Comment by susiesolis** [comment]
> @susiesolis can you also add how someone views a raw file within a particular page in GH too?

@vancluever - updated it. Can you verify that it looks okay?

File: content/source/docs/cloud/sentinel/manage-policies.html.md [hunk 1]
```diff
@@ -89,6 +89,9 @@ policy "aws-cis-4.1-networking-deny-public-ssh-acl-rules" {
 
 -> **NOTE:** At this time, only HTTP/HTTPS remote sources are supported.
 
+-> **NOTE:** Using the URL of the raw content of the policy is required when using Github. Using a URL like
+"https://github.com/hashicorp/terraform-foundational-policies-library/master/cis/aws/networking/aws-cis-4.1-networking-deny-public-ssh-acl-rules/aws-cis-4.1-networking-deny-public-ssh-acl-rules.sentinel" will give a hard error when the policy is checked. To access the Raw URL select the Sentinel file that you wish to link to in the Github repository. Once the Sentinel file is open, right click on the Raw button on the top right of the page, and save the link address.
```
Comment (vancluever) [inline content/source/docs/cloud/sentinel/manage-policies.html.md:93]:
```suggestion
`https://github.com/hashicorp/terraform-foundational-policies-library/master/cis/aws/networking/aws-cis-4.1-networking-deny-public-ssh-acl-rules/aws-cis-4.1-networking-deny-public-ssh-acl-rules.sentinel` will give a hard error when the policy is checked. To access the raw URL, select the Sentinel file that you wish to link to in the Github repository. Once the Sentinel file is open, right click on the **Raw** button on the top right of the page, and save the link address.
```

**Review by vancluever — CHANGES_REQUESTED** [review]
Just a couple of minor edits and it should be good!

**Review by justinretzolk — APPROVED** [review]
After the previously suggested changes, this looks good to me!

**Pull request merged**